src/contracts/mixins/Claiming.sol

```diff
@@ -184,6 +205,7 @@ abstract contract Claiming is ClaimingInterface, VestingInterface, IERC20 {
     /// @param account The user for which the claim is performed.
     /// @param amount The full amount claimed by the user after vesting.
     /// @param payer The address that pays the amount required by the claim.
+    /// @param sentNativeTokens Amount of ETH sent along to the transaction.
     function claimGnoOption(
         address account,
         uint256 amount,
```

Comment:
I see a whole bunch of new parameters added to the doc string but not to the function...

Reply:
The parameters were already present in the functions, but the linter didn't spot it. [Here](https://github.com/gnosis/cow-token/pull/38#issuecomment-1017219026) another comment on the topic.